owner/manager.go

```diff
@@ -65,6 +69,25 @@ const (
 	keyOpDefaultTimeout = 5 * time.Second
 )
 
+// OpType is the owner key value operation type.
+type OpType byte
+
+// List operation of types.
+const (
+	OpNone              OpType = 0
+	OpGetUpgradingState OpType = 1
```

Comment:
Can we simplify it to "OpUpgradingCluster"?

Reply:
This operation only means that the owner has received upgrading information, we cannot say that the cluster has received this information or other information